### PR TITLE
automate-chef-io: improve ldap docs

### PR DESCRIPTION
**Pull request by srenatus**
_Note: These are docs only changes, there is no associated implementation._

### :nut_and_bolt: Description: What code changed, and why?

~This doesn't do what #802 said -- because I think we already had that information on the web site all along.~ However, to make it easier to find and consume, this

- make the `ldapsearch` queries more prominent when the different phases are described
- links from the phases to their corresponding toubleshoot sections
- moves the LDAP section below _Configuration_, since it has very little to with _Authorization_
- moves the annotated config blob into `LDAP.md` (from `configuration.md`)

Also this
- removes a remark about SAML being simple (because I couldn't keep a straight face with that in our docs)
- fixes some warning shortcode markup issues

because I had happend to edit nearby sections, or noted these as worth fixing 😉 

### :chains: Related Resources

- #802 

### :+1: Definition of Done

TBD

### :athletic_shoe: How to Build and Test the Change

- `make serve` in `components/automate-chef-io`
  - if you have no github token handy, replace the `https://` url of github in `components/automate-chef-io/Makefile` by `git@github.com:chef/chef-hugo-theme`
- go to http://localhost:1313

### :white_check_mark: Checklist

- [ ] ~Tests added/updated?~
- [X] Docs added/updated?

### :camera: Screenshots, if applicable

New place for the section:

Screenshot 2019-08-08 at 14 57 20

Fixed warning box markup:

Screenshot 2019-08-08 at 14 57 43

### CODE REVIEW

File: components/automate-chef-io/content/docs/ldap.md [hunk 1]
```diff
@@ -25,9 +25,9 @@ Once the user has provided a username and password at the login screen, Chef
 Automate goes through a sequence of operations to complete the login: Connect,
 Bind, User Search, Login Bind, and Group Search.
 
-### Authorization with LDAP
+#### Authorization with LDAP
 
-Chef Automate supports using your LDAP server to authenticate users and define their permissions based on [LDAP group memberships]({{< ref "authorization-overview.md" >}}).
+Chef Automate supports defining permissions for [LDAP users and their groups]({{< ref "authorization-overview.md" >}}).
```
Comment (susanev) [inline components/automate-chef-io/content/docs/ldap.md]: is there an anchor link we can link to instead? it looks like no, but thought i would ask.

Comment (srenatus) [reply components/automate-chef-io/content/docs/ldap.md]: I'll look into it. Maybe a brief extra paragraph wouldn't hurt.... 🤔

Comment (srenatus) [reply components/automate-chef-io/content/docs/ldap.md]: "Subjects" makes sense to link to, I think. Changed it accordingly.

File: components/automate-chef-io/content/docs/configuration.md [hunk 2]
```diff
@@ -251,10 +251,10 @@ Chef Automate comes with a default LDAP configuration for Microsoft Active Direc
 The MSAD configuration is intended for getting started with minimal configuration for standard MSAD systems.
 Default values in this MSAD configuration can be overridden.
 
-{{< warning >}}
+{{% warning %}}
```
Comment (msorens) [inline components/automate-chef-io/content/docs/configuration.md]: Is that just changing a style or is it changing something incorrect to be correct? Is this Hugo syntax, or something else...?

Comment (srenatus) [reply components/automate-chef-io/content/docs/configuration.md]: The < variant takes its content verbatim and doesn't render it. This content has markup, so it needs rendering. It's Hugo, the word for it is "shortcode".

Comment (srenatus) [reply components/automate-chef-io/content/docs/configuration.md]: https://github.com/chef/automate/commit/f9b3a450294930910b24c6da042e1e20fa9794fd has had that info, too :wink:

**Comment by susanev** [comment]
@mjingle no rush at all on this, please take however much time you need.

**Comment by srenatus** [comment]
@mjingle thank you!

meta -- @sr on github isn't me :wink:

----
👋 hi @sr, sorry for the noise -- I'm using `@sr` as handle in our internal company slack, so this happens every now and then. It didn't matter much when the repo we had been working on was private, but now, you might be pinged sometimes. 😄

**Comment by msorens** [comment]
This has been incorporated as a base for another PR (1269) by docs team.

**Pull request closed without merging**